### DISCUSSION TITLE
CDC + MSC STM32L4

### DISCUSSION THREAD

**Discussion by jadczakoff** [hathach/tinyusb#1736] [Q&A] [answered]
Hello,

USB CDC doesnt't work properly because it appears on device manager but if i want send some messages, it has no connection. USB MSC not works at all. I'm using this board: 32L476GDISCOVERY. I'm using full speed mode.
link to my repo: https://github.com/jadczakoff/Test_USB_Tiny_L4
Please give me hint where is the problem.

**Comment by hathach** [comment, marked as answer]
which dcd driver you are using ? you should use `synopys/dwc2` instead of the st/synopsys driver

**Reply by jadczakoff** [reply]
Thanks for help, now it's working :D

**Reply by PaulNewf** [reply]
Please ignore request, Just realized your project is available here on Github
-Paul
